pytrendy/post_processing/segments_refine.py

```diff
@@ -787,7 +790,7 @@ def fill_in_flats(segments:list):
     return segments_refined
 
 
-def refine_segments(df: pd.DataFrame, value_col: str, segments: list, method_params:dict):
+def refine_segments(df: pd.DataFrame, value_col: str, segments: list, method_params: dict) -> list:
```

Comment:
In the refine segments file it is worth explicitly making the return type a list of dicts. When returning segments or refined_segments, good to set it to list of dicts explicitly instead of just vague list